Makefile

```diff
@@ -79,11 +81,15 @@ install:
 testbuild: TESTS := $(shell $(GO) list -tags '$(TAGS)' $(PKG))
 testbuild: GOFLAGS += -c
 testbuild:
+	ROOT="$(DIR)"; \
 	for p in $(TESTS); do \
 	  NAME=$$(basename "$$p"); \
 	  OUT="$$NAME.test"; \
-	  DIR=$$($(GO) list -f {{.Dir}} -tags '$(TAGS)' $$p); \
-	  $(GO) test $(GOFLAGS) -tags '$(TAGS)' -o "$$DIR"/"$$OUT" -ldflags '$(LDFLAGS)' "$$p" $(TESTFLAGS) || exit 1; \
+	  DIRROOT=$${ROOT-$$($(go) list -f {{.Root}} -tags '$(TAGS)' $$p)}; \
```

Comment:
it is a bit tedious, but having an external script makes passing all the various make options hard.
What I could do is keep `testbuild` simple. eg: no stripping, no custom output dir, maybe even a single package, and instead have separate script which does all test binaries stripped in a custom directories. the only parameter would be that directory. However, any changes would have to be synced between the two.
An in-between solution would be to keep testbuild simple and have a testbuildstatic with separate defaults.

And fixed the lowercase go, that was a typo. worked though, surprisingly.